index.html

```diff
@@ -0,0 +1,316 @@
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+                </section>
+
+                <section class="work-cards-row">
+                    <!-- card1 -->
+                    <div class="card">
+                        <section class="card-content">
+                            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+                            
+                            <ul class="card-tags">
+                                <li class="card-tag ruby"><p class="card-tag-text">Ruby on rails</p></li>
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+
+                    <!-- card2 -->
+                    <div class="card">
+                        <section class="card-content">
+                            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+                            
+                            <ul class="card-tags">
+                                <li class="card-tag ruby"><p class="card-tag-text">Ruby on rails</p></li>
+                                <li class="card-tag css"><p class="card-tag-text">css</p></li>
+                                <li class="card-tag Javascript"><p class="card-tag-text">Javascript</p></li>
+                                <li class="card-tag html"><p class="card-tag-text">html</p></li>
+                            </ul>
+
+                            <div class="card-title-button">
+                                <button class="card-button"><p class="card-button-text">See Project</p></button>
+                            </div>
+                        </section>
+                    </div>
+
+                    <!-- card3 -->
+                    <div class="card">
+                        <section class="card-content">
+                            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
```

Comment:
Kindly use an h3 here as it is directly under an h2. it obeys the hx or header hierarchy rule and it is semantic hence will aid user accessibility for screen readers😉

Reply:
Fixed.